recipes/spectra/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "spectra" %}
+{% set version = "0.0.7" %}
+{% set sha256 = "a4053ea70d437fd0ab5afb9e4a5295b400b53681c4a6c990b2a0dc7659f68b5a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - colormath
```

Comment:
Is `colormath` needed at build time? If not, could you please drop it from `build`?